admin-cli/lib/cut.ts

```diff
@@ -80,8 +81,10 @@ export function cut(versionOrType, dryRun) {
   } else {
     return concat(
       generateChangelog({ dryRun, version: tagVersion }),
-      of(updatePackageJsons(versionWithBuild)),
```

Comment:
@PascalPrecht 

Notice that `concat(obs1, of(runSomeCodeImmediately())` doesn't make much sense or at least I can't think of a good use case.

Reply:
To elaborate (because I was in a hurry earlier). Usually you use `concat(obs1, afterPrevious)` with the intention to run `afterPrevious` when `obs1` finishes. But that only makes sense if `afterPrevious` really is a lazy Observable (Observables are always lazy by default). If you do something like `concat(obs1, of(Date.now())` then obviously the date will the date from the time where the Observable pipeline was constructed and not from the time that `obs1` completed and the other observable was subscribed to. So, in this case you really want `defer` to have a lazy Observable to work with.

Reply:
Thanks for pointing this out. I ran the command locally and it seemed everything worked as expected. While it did output as expected it didn't generate additional changelogs, but I also tested the command on a commit history with no new changes, so I thought that's just what generate-changelog does.

Sorry for messing this up